lib/currency-utils.js

```diff
@@ -56,3 +41,11 @@ export function formatCurrency(amount, currency = 'USD', options = {}) {
 
 export const formatValueAsCurrency = (value, options) =>
   formatCurrency(value.valueInCents || value.value * 100, value.currency, options);
+
+export const floatAmountToCents = floatAmount => {
```

Comment:
I took the liberty of moving this out of `lib/math`, `lib/currency-utils` feels like a better place to have this